packages/babel-plugin-transform-modules-commonjs/test/fixtures/regression/issue-9611/input.mjs

```diff
@@ -0,0 +1,3 @@
+import x from './x';
+
+export default function(x) {}
```

Comment:
Looking at the code I would guess it throws, because on line 465 it tries to get the id of the declaration but this one does not have one. but maybe path.get() does never return null?

Reply:
It is set at https://github.com/babel/babel/blob/47bee79acf39e4fba0e21c4e2c52ea6cd11fce67/packages/babel-helper-split-export-declaration/src/index.js#L34

Reply:
👍